WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListItemUiStateHelper.kt

```diff
@@ -196,9 +203,11 @@ class PostListItemUiStateHelper @Inject constructor(private val appPrefsWrapper:
         }
     }
 
-    private fun shouldShowOverlay(uploadStatus: PostListItemUploadStatus): Boolean {
+    private fun shouldShowOverlay(uploadStatus: PostListItemUploadStatus, performingAction: Boolean): Boolean {
         // show overlay when post upload is in progress or (media upload is in progress and the user is not using Aztec)
-        return uploadStatus.isUploading || (!appPrefsWrapper.isAztecEditorEnabled && uploadStatus.isUploadingOrQueued)
```

Comment:
I'd consider removing the comment and

creating a new variable for `(!appPrefsWrapper.isAztecEditorEnabled && uploadStatus.isUploadingOrQueued)` -> something like `val uploadingMediaAndEditingNotSupported = (!appPrefsWrapper.isAztecEditorEnabled && uploadStatus.isUploadingOrQueued)`...

or

replacing `uploadStatus.isUploadingOrQueued` with `hasInProgressMediaUpload`. We can do it in another PR as it's unrelated and it'll be easier to follow up if it introduces a bug.